src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -925,15 +876,35 @@ test_run_mb(void)
  * TODO: Fix those eventually.
```

Comment:
Actually, I think there is nothing to fix here, its just that we cannot use all micro-benchmark test cases for virtual-machine environments, so taking out only the FIXME.